### DISCUSSION TITLE
Restrict media folder per user

### DISCUSSION THREAD

**Discussion by DeMiro5001** [joomla/joomla-cms#43950] [Q&A] [answered]
I plan to deploy a joomla website for a team. Every user has access to his articles only, made possible using different categories for each one.
But I was unable to have uploaded files for each user accessible only by him or by users with higher rights.
I don't know if there is an override that can help me or even if I can define a configuration.php per user or define the media folder as images/$username or any other idea.
Thanks a lot for your help

**Comment by brianteeman** [comment, marked as answer]
https://restrictedfs.dgrammatiko.dev/

**Reply by DeMiro5001** [reply]
I will check that, thanks a lot

**Reply by DeMiro5001** [reply]
OK, I tried to install it but joomla 5 said that it can't find the manifest. I extracted the content of the archive with 7zip then recompressed it to zip. The new file succeeded installation. I will inform the maintainer of the plugin.
